containers/kubernetes/how-to/enable-easy-deploy.mdx

```diff
@@ -76,11 +76,11 @@ You can also deploy off-the-shelf applications pre-configured for Scaleway produ
       Example: 45 23 * * 6
       This example executes program at 23:45 (11:45 PM) every Saturday.
       ```
-      You must also include a path to your script in the **Command** box and set up environment variables. 
+      You must also include a path to your script in the **Command** box and set up environment variables.
 
     * Select a deployment type and fill out the prompted information according to your choice. A summary of your choices is available.
 
-    *  Click **Deploy an Application** to proceed. The list of your deployed applications displays. 
+    *  Click **Deploy an Application** to proceed. The list of your deployed applications displays.
```

Comment:
"Deploy an Application" or "Deploy application" (as seen below) ?

Reply:
the second one. I'll fix it. Thanks!